app/models/offer.rb

```diff
@@ -130,6 +130,7 @@ def set_initial_state
 
     before_transition on: :submit do |offer, _transition|
       offer.submitted_at = Time.now
+      offer.created_by.update_attribute(:sms_reminder_sent_at, Time.now + 1.minute) # start the SMS reminder clock from here
```

Comment:
@namrataukirde using Time.now works correctly - I've tested this on live.

Reply:
Great.. Thanks Steve for checking it. :)